### PR TITLE
[feat] Added alerter component!

### PR DESCRIPTION
**Pull request by y-lohse**
A basic component to display notifications!

It comes in 3 variations: error, success and info.
It auto-closes after a few seconds or you can provide a button text and callback to handle the closing elsewhere.

### CODE REVIEW

File: stylus/ui-components/alerts.styl [hunk 1]
```diff
@@ -30,6 +29,7 @@ $alerts
 
     .coz-alert
         display          block
+        z-index          100
```
Comment (GoOz) [inline stylus/ui-components/alerts.styl]: What changed since the last version of alerter that explains this new z-index? Unless it's a bugfix, maybe?

Comment (y-lohse) [inline stylus/ui-components/alerts.styl]: As far as I can tell, the other z-index in the file (`coz-alerter`) has no effect. To me it's a bugfix, but I'm not sure why the other z-index was there in the first place. I'm happy to change it back if it has implications I didn't see.

Comment (GoOz) [inline stylus/ui-components/alerts.styl]: Honestly I don't know, I'm not the one who wrote it. Maybe a cleaning would be a good thing but not right now. And I have some kind of a Z-Index manager in my todo list, so we'll see that later.
So, if it's a bugfix then I'm good :)

**Comment by goldoraf** [comment]
:tada:

**Pull request merged**